pkg/certificate/analyzer.go

```diff
@@ -62,26 +62,29 @@ func (a *analyzer) start() {
 			// cap the list to an arbitrary amount (hopefully smaller than the 16 MB document size cap)
 			// anything approaching this limit will cause performance issues in software that depends on rita
 			// anything tuncated over this limit won't be visible as an IP connecting to an invalid cert
-			if len(datum.OrigIps) > 200003 {
-				datum.OrigIps = datum.OrigIps[:200003]
+			origIPs := datum.OrigIps.Items()
+			if len(origIPs) > 200003 {
+				origIPs = origIPs[:200003]
 			}
 
-			if len(datum.Tuples) > 20 {
-				datum.Tuples = datum.Tuples[:20]
+			tuples := datum.Tuples.Items()
```

Comment:
Note that `.Items()` must be called in order to convert the map into an array. 
https://github.com/activecm/rita/commit/c4b5b3fa83b503af6da624a9727cbf1bfbaaa45a